e/erubi/erubi_ubi8.4.sh

```diff
@@ -0,0 +1,64 @@
+# -----------------------------------------------------------------------------
+#
+# Package	: erubi
+# Version	: v1.10.0
+# Source repo	: https://github.com/jeremyevans/erubi
+# Tested on	: UBI 8.4
+# Language      : Ruby
+# Travis-Check  : False
+# Script License: Apache License, Version 2 or later
+# Maintainer	: Sapana Khemkar {Sapana.khemkar@ibm.com}
+#
+# Disclaimer: This script has been tested in root mode on given
+# ==========  platform using the mentioned version of the package.
+#             It may not work as expected with newer versions of the
+#             package and/or distribution. In such case, please
+#             contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+
+set -e 
+
+PACKAGE_NAME=erubi
+PACKAGE_VERSION=1.10.0
+PACKAGE_URL=https://github.com/jeremyevans/erubi
+
+yum install -y nodejs nodejs-devel nodejs-packaging git wget curl ruby ruby-devel rubygem-rake procps libcurl-devel libffi-devel sqlite sqlite-devel
+
+yum install -y --allowerasing gcc gcc-c++ yum-utils make automake autoconf libtool gdb* binutils rpm-build gettext wget
+
+yum-config-manager --add-repo http://mirror.centos.org/centos/8/AppStream/ppc64le/os/ && yum-config-manager --add-repo http://mirror.centos.org/centos/8/PowerTools/ppc64le/os/ && yum-config-manager --add-repo http://mirror.centos.org/centos/8/BaseOS/ppc64le/os/ && yum-config-manager --add-repo http://mirror.centos.org/centos/8/virt/ppc64le/ovirt-44/
```

Comment:
I know this is coming from the template but is there a way to avoid enabling the centos repositories for this one ? are all the dependencies being installed needed for example?

Reply:
Have cleaned up and removed installation of unwanted packages